server_addon/aftereffects/client/ayon_aftereffects/api/workfile_template_builder.py

```diff
@@ -92,7 +92,7 @@ def _get_item(self, placeholder_item):
         return None, None
 
     def _collect_scene_placeholders(self):
-        """" Cache placeholder data to shared data.
+        """ Cache placeholder data to shared data.
```

Comment:
```suggestion
        """Cache placeholder data to shared data.
```

Reply:
Thanks, wasn't sure whether that was some personal taste of somene. I saw a lot of this elsewhere - I think it was in the Nuke integration? Not sure how pissed you get by these spaces but if you need me I can make a separate cosmetic PR for those - hehe.